src/components/ExploreNav/styled.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import { BLACK, H3, CREAM, PINK } from "../../styles/styles";
+import { min } from "../../styles/breakpoints";
+import Cursor from "../../images/cursor-pointer.svg";
+import { AnchorLink } from "gatsby-plugin-anchor-links";
+
+export const StyledLink = styled.div`
+  white-space: nowrap;
+  height: 3rem;
+  width: 100%;
+  transform: skew(15deg);
+  background-color: transparent;
+  cursor: url(${Cursor}) 20 20, pointer;
+  border: 2px solid ${CREAM};
+  text-align: center;
+  &:hover {
+    background-color: ${PINK};
+  }
+`;
+
+export const LeftLink = styled(StyledLink)`
+  border-right: none;
+`;
+
+export const CenterLink = styled(StyledLink)`
+  border-right: none;
+  border-left: none;
+`;
+
+export const RightLink = styled(StyledLink)`
+  border-left: none;
+`;
+
+export const LinkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-left: 5%;
+  width: 90%;
+  ${min.tablet} {
+    margin-left: 25%;
+    width: 50%;
+  }
+  ${min.desktop} {
+    width: 33.33%;
+    margin-left: 33.33%;
+  }
+`;
+
+export const LinkText = styled(H3)`
+  margin: 0.6rem 0;
+  transform: skew(-15deg);
+  color: ${CREAM};
+  &:hover {
+    color: ${BLACK};
```

Comment:
can you move this to the `StyledLink` where you set the background color to pink? right now if you hover over the button but not the text itself it remains white

Reply:
for some reason when setting it in the StyledLink it isn't changing the color of the text, but I added a height and width to the LinkText and it seems to have solved the problem

Reply:
sounds good! I think adding `color: inherit` here would have also worked